hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/ProcessFailureExitHandler.java

```diff
@@ -0,0 +1,13 @@
+package org.apache.mesos.hdfs.executor;
+
+import org.apache.mesos.hdfs.util.HDFSConstants;
+import org.apache.mesos.hdfs.util.FailureUtils;
+
+/**
+ * When a process fails this handler will throw a runtime exception.
+ */
+public class ProcessFailureExitHandler implements ProcessFailureHandler {
```

Comment:
ProcessFailureExitHandler is an HDFS thing and should probably be named such.

probably HDFSProcessExitHandler or ?